docker-compose-local.yml

```diff
@@ -19,14 +19,6 @@ services:
     tty: true
     depends_on:
       - jsonserver
-    develop:
```

Comment:
please don't delete this

Reply:
Sorry about that! Copy! Will put it back now and won't edit this next time!